wheel_mps_mixer:
  ros__parameters:
    input_topic: /cmd_vel
    output_topic: /wheel_cmd

    front_track_width: 0.75
    rear_track_width: 1.17

    turn_gain_front: 0.7
    turn_gain_rear: 1.0

    front_left_scale: 1.0
    front_right_scale: 1.0
    rear_left_scale: 1.0
    rear_right_scale: 1.0

    publish_rate_hz: 50.0
    cmd_timeout_ms: 300

    max_v: 1.0
    max_w: 1.0
    max_wheel_mps: 1.5

    max_accel_mps2: 0.8

    dir_fl: 1
    dir_fr: 1
    dir_rl: 1
    dir_rr: 1
